.github/workflows/build.yml

```diff
@@ -35,19 +35,18 @@ jobs:
           rust-wasm: true
           rust-cache: true
 
-      - name: Cargo Format
+      - name: Run lints
         run:
-          BUILD_SPIN_EXAMPLES=0 cargo fmt --all -- --check
+          BUILD_SPIN_EXAMPLES=0 make lint
 
-      - name: Cargo Clippy
-        run:
-          BUILD_SPIN_EXAMPLES=0 cargo clippy --workspace --all-targets --features all-tests -- -D warnings
+      - name: Cancel everything if this fails
```

Comment:
Super optional nit: `Cancel everything if lint fail`

Reply:
👍 I'll add this to another PR so I don't need to run CI again just for that.